components/plurals/benches/operands.rs

```diff
@@ -27,7 +27,11 @@ fn operands(c: &mut Criterion) {
                     .expect("Failed to parse a number into an operands.");
             }
             for s in &data.fixed_decimals {
-                let f: FixedDecimal = FixedDecimal::from(s.value).multiplied_pow10(s.exponent);
+                let f: SignedFixedDecimal = {
+                    let mut dec = SignedFixedDecimal::from(s.value);
+                    dec.multiply_pow10(s.exponent);
```

Comment:
Nit: `multiply_pow10` should be on `dec.absolute`, right?

Reply:
done